airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -127,3 +144,77 @@ def run_query(self, query_id: str, params: Dict[str, Any]) -> None:
             .runquery(queryId=query_id, body=params)
             .execute(num_retries=self.num_retries)
         )
+
+    def create_sdf_download_operation(self, body_request: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Creates an SDF Download Task and Returns an Operation.
+        :param body_request: Body request.
+        :type body_request: Dict[str, Any]
+
+        More information about body request n be found here:
+        https://developers.google.com/display-video/api/reference/rest/v1/sdfdownloadtasks/create
+
+        """
+        request = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .sdfdownloadtasks()
+            .create(body_request=body_request)
+            .execute(num_retries=self.num_retries)
+        )
+        return request["name"]
+
+    def get_sdf_download_operation(self, name: Dict[str, Any]):
+        """
+        Gets the latest state of an asynchronous SDF download task operation.
+        :param name: The name of the operation resource.
+        :type name: str
+        """
+
+        request = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .sdfdownloadtasks()
+            .operation()
+            .get(name=name)
+            .execute(num_retries=self.num_retries)
+        )
+        return request
+
+    def download_media(self, resource_name: str, filename: str):
+        """
+        Downloads media.
+        :param resource_name:  of the media that is being downloaded.
+        :type resource_name: str
+        :param filename: object used for saving result of operation get_sdf_download_tasks
+        """
+
+        request = (
+            self.get_conn_to_display_video()  # pylint: disable=no-member
+            .media()
+            .download_media(resource_name=resource_name)
+        )
+        with open(filename, "w+") as target_file:
+            downloader = MediaIoBaseDownload(target_file, request, chunksize=1024 * 1024)
+            done = False
+            while done is False:
+                status, done = downloader.next_chunk()
+                if status:
+                    self.log.info("Download %d.", int(status.progress() * 100))
+            self.log.info("Download Complete!")
+
+    def wait_for_download_operation(self, operation: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def _wait_for_download_operation(self, operation: Dict[str, Any]) -> Dict[str, Any]:
```

Reply:
The name and signature are misleading imho. I would expect to get bool value or None (pass or rise exception)